buildkite/scripts/single-node-tests.sh

```diff
@@ -2,27 +2,23 @@
 
 set -eo pipefail
 
-if [[ $# -ne 1 ]]; then
-    echo "Usage: $0  <path-to-source-tests>"
-    exit 1
-fi
+# Don't prompt for answers during apt-get install
+export DEBIAN_FRONTEND=noninteractive
 
-path=$1
+sudo apt-get update
+sudo apt-get install -y git apt-transport-https ca-certificates tzdata curl python3 python3-pip wget
 
-eval "$(opam config env)"
-export PATH="/home/opam/.cargo/bin:/usr/lib/go/bin:$PATH"
-export GO=/usr/lib/go/bin/go
+git config --global --add safe.directory /workdir
 
-# TODO: Stop building lib_p2p multiple times by pulling from buildkite-agent artifacts or docker or somewhere
-echo "--- Build libp2p_helper TODO: use the previously uploaded build artifact"
-make -C src/app/libp2p_helper
+source buildkite/scripts/export-git-env-vars.sh
```

Comment:
Just a though: I guess this won't work for forks until you merge #14596

Reply:
yes, However we are planning to merge a couple commits from rampup to  berkeley soon